ydb/core/kafka_proxy/actors/control_plane_common.h

```diff
@@ -0,0 +1,376 @@
+#include "ydb/core/kafka_proxy/kafka_constants.h"
+
+#include "ydb/core/grpc_services/base/base.h"
+
+#include "ydb/core/kafka_proxy/actors/actors.h"
+
+#include "ydb/core/kafka_proxy/kafka.h"
+
+#include <ydb/core/kafka_proxy/kafka_events.h>
+
+#include <ydb/services/lib/actors/pq_schema_actor.h>
+
+namespace NKafka {
+
+struct TRetentionsConversionResult {
+    bool IsValid = true;
+    std::optional<ui64> Ms;
+    std::optional<ui64> Bytes;
+
+    std::vector<TString> ErrorMessages;
+
+    TAutoPtr<TEvKafka::TEvTopicModificationResponse> GetKafkaErrorResponse(TString topicName) {
+        Y_ABORT_IF(IsValid);
+        Y_ABORT_UNLESS(ErrorMessages.size() > 0);
+
+        auto reponse = MakeHolder<TEvKafka::TEvTopicModificationResponse>();
+        reponse->TopicPath = topicName;
+        reponse->Status = INVALID_CONFIG;
+        TString resultErrorMessage = ErrorMessages.size() == 1
```

Comment:
Is it gurantied that ErrorMessages has 1 or 2 elements?

Reply:
Yes, now it is.
But I changed the code to support any number of elements anyway.